packages/vaadin-combo-box/test/object-values.test.js

```diff
@@ -83,9 +84,10 @@ describe('object values', () => {
       selectItem(0);
 
       comboBox.itemLabelPath = 'custom';
+      comboBox.opened = true;
```

Comment:
Out of curiosity, does this change in tests here and in other test where you added `comboBox.open()` mean that virtualizer gets initialized lazily and it is different compared to what we had with `iron-list`?

If so, we could probably consider removing the `disable-upgrade` hack as it might not be needed anymore, and that should also make moving to Lit a bit easier later on.

Reply:
We could look into removing the hack. In this case, the actual `<vaadin-combo-box-item>` elements are already in the DOM, but they don't get updated until the overlay is opened (which kind of makes sense).

Reply:
Got it, thanks.